app/src/ui/business-card/BusinessCard.module.scss

```diff
@@ -0,0 +1,209 @@
+@import "src/theme/base";
+
+$card-border: #e6ebf0;
+$card-background: #ffff;
+$card-subtitle: #898f99;
+
+$card-width: 253px;
```

Comment:
This var should not be set. The right way of setting the width is by using the `Grid.Col lg={3}` component. Width should be left to 100% and be adjusted automatically depending on the Column width.